src/common/auth/actions.js

```diff
@@ -39,6 +47,11 @@ export function login(fields) {
     payload: {
       promise: validateForm(validate, fields)
         .then(() => post(fetch, 'auth/login', fields))
+        .then(response => {
+          credentialsStore.set('authToken', response.authToken);
+          dispatch(setAuthToken(response.authToken));
```

Comment:
We can get rid of this. I see two ways:
1) Dispatch it from the component after `login` action is finished. But then we need some way how to pass back the auth token to the component from the action (that means in my opinion breaking unidirectional data flow). Or we could rely on the `injectDependencies` middleware, but then I do not know how to fire it when rendering on server.
2) We could set it in the auth reducer, but then there would be some code duplication because we still need the `setAuthToken` action IMO